DeepCat.css

```diff
@@ -13,3 +13,40 @@
     background-repeat: no-repeat;
     background-size: 24px auto;
 }
+
+#deepcat-hintbox, #deepcat-smallhint {
+    background: #f3f3f3;
+    border: 1px solid silver;
+    font-size: 13px;
+}
+
+#deepcat-hintbox {
+    padding: .75em;
+    margin-top: 1em;
+    margin-bottom: 1em;
+    width: 38em;
```

Comment:
I copied what was there for div.searchresult. They are fixed to this width and we want the box to have the same width.